lib/karp/catalog/video.ts

```diff
@@ -21,9 +21,8 @@ export function decodeVideo(o: unknown): o is Video {
 	const obj = o as Partial<Video>
 	if (!decodeTrack(obj.track)) return false
 	if (typeof obj.codec !== "string") return false
-	if (typeof obj.description !== "string") return false
 
-	obj.description = Hex.decode(obj.description)
+	obj.description = obj.description && typeof obj.description === "string" ? Hex.decode(obj.description) : undefined
```

Comment:
I did this instead, it runs and also passes build and biome. Tested with both fMP4/AVCC and annexb